Modified OMALS submission form

kdweeks opened the pull request:
@alawvt @pyc1 I have made the changes to the OMALS submission form. I have attached a screenshot of the "Disciplines" and "Items Type" sections of the form. Please review them.  

<img width="968" alt="OMALS_Disciplines" src="https://user-images.githubusercontent.com/473202/69989939-bc3e1000-1512-11ea-92c7-6100ff2cb3d7.png">


<img width="968" alt="OMALS_Items1" src="https://user-images.githubusercontent.com/473202/69989952-c3651e00-1512-11ea-9cc1-1b76fb7e955e.png">


<img width="968" alt="OMALS_Items2" src="https://user-images.githubusercontent.com/473202/69989960-c829d200-1512-11ea-88a5-4e8517853aee.png">


<img width="968" alt="OMALS_Items3" src="https://user-images.githubusercontent.com/473202/69989974-cc55ef80-1512-11ea-89eb-21c54b73fb05.png">

This pull request resolves issue #657 

This pull request also resolves issue #639

kdweeks committed:
Modified OMALS submission form

alawvt commented:
@kdweeks, please remove the discontinued options from the OMALS list.

`Learning object` should be lower case for second word.

Make content type a required field.

kdweeks committed:
Updated OMALS submission Form

kdweeks commented:
I have updated the OMALS submission form to:

Change the dc.type form field to be required.
Change the dc.type value from "Learning Object" to "Learning object."
Remove the discontinued values from the discipline field.

I have updated the OMALS submission form to:
- Change the dc.type form field to be required.
- Change the dc.type value from "Learning Object" to "Learning object."
- Remove the discontinued values from the discipline field.

Screenshots are attached.  @alawvt and @pyc1, Please review at your convenience.

<img width="1183" alt="OMALS_screenshot1" src="https://user-images.githubusercontent.com/473202/70089809-ef51d380-15e6-11ea-87cd-1fb40b696832.png">

<img width="1183" alt="OMALS_screenshot2" src="https://user-images.githubusercontent.com/473202/70089826-f5e04b00-15e6-11ea-81b2-342116644907.png">

<img width="1183" alt="OMALS_screenshot3" src="https://user-images.githubusercontent.com/473202/70089841-fd075900-15e6-11ea-8c7a-5b4ffd26c38c.png">

<img width="1183" alt="OMALS_screenshot4" src="https://user-images.githubusercontent.com/473202/70089863-04c6fd80-15e7-11ea-9026-8eea9dd2fc05.png">
